manifests/dynamic/kruize-ui-nginx-setup.yml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
All yaml files need to use the suffix `.yaml` and not `.yml` in line with the convention that we are following in all the other repos

Reply:
changed it to yaml